org.gephi.Gephi.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.gephi.Gephi
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/gephi
+
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+
+modules:
+  - name: gephi
+    buildsystem: simple
+    build-commands:
+      - tar -xf gephi.tar.gz
+      - rm -rf gephi.tar.gz
+      - mv gephi-*/{bin,etc,extra,gephi,jre-x64,platform} /app
+    sources:
+      - type: file
+        url: https://github.com/gephi/gephi/releases/download/v0.9.7/gephi-0.9.7-linux-x64.tar.gz
+        sha256: 082c4015454ac0ac52ec50b4c97ed7db7810007e18db7beb05e65a11ac19a9c5
+        x-checker-data: {
+            type: json,
+            url: https://api.github.com/repos/gephi/gephi/releases/latest,
+            version-query: .tag_name | gsub("v";""),
+            url-query: '"https://github.com/gephi/gephi/releases/download/v" + $version + "/gephi-" + $version + "-linux-x64.tar.gz"'
+        }
+        dest-filename: gephi.tar.gz
+        only-arches:
+          - x86_64
```

Comment:
that doesn't prevent building on aarch64.

Also since it seem to be Java, why excluding aarch64? Runtimes are available for both.

Reply:
We rely on some OpenGL library that is currently not working properly on this architecture so we want to only release for x64 for now.

Reply:
You need to add a file named `flathub.json` as described at https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
Got it, I missed that it had to be a separate file. I added it now.